src/components/course/CourseMaterialsButton.jsx

```diff
@@ -30,7 +30,7 @@ const CourseMaterialsButton = ({ className }) => {
       className={className}
       variant="brand"
       as={Hyperlink}
-      destination={userEnrollment.resumeCourseRunUrl || userEnrollment.linkToCourse}
+      destination={userEnrollment.linkToCourse}
```

Comment:
[inform] `.linktoCourse` already takes into account `resumeCourseRunUrl`. See https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/cf529d9394e5258e846aaad51c986223c74721d9/src/components/app/data/utils.js#L303-L307 for more details.